tests/integration/cmd_pref_config_test.go

```diff
@@ -262,6 +263,23 @@ var _ = Describe("odo preference and config command tests", func() {
 			Expect(configValue).To(Not(ContainSubstring(("PORT"))))
 			Expect(configValue).To(Not(ContainSubstring(("SECRET_KEY"))))
 		})
+
+		It("should be able to unset an environment variable with foo=bar", func() {
+
+			// Create the project
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", project, "--context", context)
+
+			// Test out passing in with an equals sign
+			helper.CmdShouldPass("odo", "config", "set", "--env", "FOOBAR=foobar", "--context", context)
+			output := helper.CmdShouldPass("odo", "config", "view", "--context", context)
+			Expect(output).To(ContainSubstring("FOOBAR"))
+
+			// Check that we can remove it
+			helper.CmdShouldPass("odo", "config", "unset", "--env", "FOOBAR=foobar", "--context", context)
```

Comment:
I dont think someone needs to provide the value of the env var as well to unset it? something like `odo config unset --env FOOBAR` should also work

Reply:
That's the point of the test. To test to see if equals still works ex. `FOOBAR=qwerty` would unset FOOBAR.